api_tests/base/test_serializers.py

```diff
@@ -76,7 +76,7 @@ def test_registration_serializer(self):
         # fields that are visible for withdrawals
         visible_on_withdrawals = ['contributors', 'date_created', 'description', 'id', 'links', 'registration', 'title', 'type', 'current_user_can_comment']
         # fields that do not appear on registrations
-        non_registration_fields = ['registrations', 'draft_registrations']
+        non_registration_fields = ['registrations', 'draft_registrations', 'templated_from']
```

Comment:
Although if you're not showing the field at all on registrations, then you don't need the lambda on https://github.com/CenterForOpenScience/osf.io/pull/6038/files#r72799108 at all